README.md

```diff
@@ -241,32 +244,37 @@ EQ_KEYS_FILE=dev-keys.yml EQ_SECRETS_FILE=dev-secrets.yml ./k8s/deploy_credentia
 
 ### Deploying the app
 
-The following environment variables can be set when deploying the app.
-- SUBMISSION_BUCKET_NAME
-- DOCKER_REGISTRY *(optional)*
-- IMAGE_TAG *(optional)*
-- GOOGLE_TAG_MANAGER_ID *(optional)*
-- GOOGLE_TAG_MANAGER_AUTH *(optional)*
-- GOOGLE_TAG_MANAGER_PREVIEW *(optional)*
-- REQUESTED_CPU_PER_POD *(optional)* - No. of CPUs to request per Pod
-- ROLLING_UPDATE_MAX_UNAVAILABLE *(optional)* - Specifies the maximum number of Pods that can be unavailable during the update process.
-- ROLLING_UPDATE_MAX_SURGE *(optional)* - Specifies the maximum number of Pods that can be created over the desired number of Pods.
-- MIN_REPLICAS *(optional)* - Minimum no. of replicated Pods
-- MAX_REPLICAS *(optional)* - Maximum no. of replicated Pods
-- TARGET_CPU_UTILIZATION_PERCENTAGE *(optional)* - The average CPU utilization usage before auto scaling applies
+The following environment variables should be set when deploying the app.
+
+| Variable Name                             | Description                                                                          |
+|-------------------------------------------|--------------------------------------------------------------------------------------|
+| PROJECT_ID                                | The ID of the GCP target project                                                     |
```

Comment:
PROJECT_ID is only required for Concourse deploys, so should only be in the CI Readme.